wsproxy.opam

```diff
@@ -0,0 +1,14 @@
+opam-version: "1.2"
+name: "wsproxy"
+maintainer: "xen-api@lists.xen.org"
+authors: [ "Jon Ludlam <jonathan.ludlam@citrix.com>"
+           "Marcello Seri <marcello.seri@citrix.com>" ]
+license: "LGPL-2.0 with OCaml linking exception"
+homepage: "https://github.com/xapi-project/wsproxy"
+dev-repo: "https://github.com/xapi-project/wsproxy.git"
```

Comment:
We could add a `tags: [ "org:xapi-project" ]` line to all our packages. That will be useful in case we upstream them.